packages/test-runner-webdriver/src/IFrameManager.ts

```diff
@@ -155,6 +155,11 @@ export class IFrameManager {
     return this.driver.keys(keys);
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async performActions(_: string, _actions: object[]) {
+    console.warn('Device actions are not supported in the concurrent mode.');
```

Comment:
I would rather explicitly mention the necessary config option, e.g. something like this:

```
Unsupported operation. In order to use device actions in Webdriver, set "concurrency" to 1.
```

Reply:
Good point, done.